handler.go

```diff
@@ -0,0 +1,74 @@
+package main
```

Comment:
let's give this file a that is specific to its resource...I'd suggest `key_handler`.

Reply:
Also, lets put this in a handler directory. You will want to make the functions public as well.

Reply:
Done!